edison.coffee

```diff
@@ -16,6 +16,10 @@ OSX_LINUX_FLASH = '''
 	Execute the following from the unzipped directory:
 	<br>
 	<code>sudo ./flashall.sh</code>
+	<br>
+	If you have trouble executing, you may need to modify the file permissions first:
+	<br>
+	<code>sudo chmod 755 ./flashall.sh</code>
```

Comment:
I think it should be `sudo chmod +x ./flashall.sh`

Reply:
Fixed